azext_iot/common/_azure.py

```diff
@@ -120,7 +120,9 @@ def _find_iot_dps_from_list(all_dps, dps_name):
 
 
 def get_iot_central_tokens(cmd, app_id, token, central_dns_suffix):
-    import requests
+    from requests import post
```

Comment:
how come we arent using `http` here?

Reply:
"http" is just a variable name. Also there is no reason to use Session here as there's just one call and verification settings can be put in the post options. Session is useful in other files where multiple requests need to share same settings